src/SearchIndexer/Sfa.Das.Sas.Tools.MetaDataCreationTool/MetaDataManager.cs

```diff
@@ -60,7 +60,13 @@ public void GenerateStandardMetadataFiles()
 
         public List<StandardMetaData> GetStandardsMetaData()
         {
-            var standards = JsonConvert.DeserializeObject<List<StandardMetaData>>(_httpService.Get(string.Concat(_appServiceSettings.MetadataApiUri, "Standard"), null, null));
```

Comment:
Hey dude, 

Can you instead of using and injecting (IHttpGet) httpService maybe create a metaDataSerivce (like vstsService) for the new metadata source. 
All we have to do in metaDataManager then is to all that service .getAllStandards or getAllFrameworks